src/Api/StructureShape.php

```diff
@@ -6,6 +6,8 @@
  */
 class StructureShape extends Shape
 {
+    protected $members;
```

Comment:
Any reason this is protected? I try to avoid protected members since they expose an implementation detail of an implementation detail to subclasses.

- https://books.google.com/books?id=ka2VUBqHiWkC&pg=PA89&lpg=PA89&dq=effective+java+protected+members&source=bl&ots=yZFkOmt-NX&sig=oTFs5pu9W45xG6wOQGBnL5nZ4l8&hl=en&sa=X&ei=QdWSVdCBMc_1oASB9KOwAw&ved=0CDoQ6AEwBA#v=onepage&q=effective%20java%20protected%20members&f=false
- http://programmers.stackexchange.com/questions/162643/why-is-clean-code-suggesting-avoiding-protected-variables